load/PreValidate.py

```diff
@@ -66,8 +83,12 @@ def validateDBPELT(lptsReader, s3Client, location, directory, fullPath):
 		stockNumber = preValidate.isTextStockNo(directory)
 		if stockNumber != None:
 			filenames = unicodeScript.getFilenames(s3Client, location, fullPath)
-			sampleText = unicodeScript.readObject(s3Client, location, fullPath + "/" + filenames[0])
-			resultList = preValidate.validateUSXStockNo(stockNumber, filenames, sampleText)
+			sampleText = unicodeScript.readObject(s3Client, location, filenames[0])
```

Comment:
related to my earlier comment about data organization (for input), our new code should not be creating directories, or changing the location of where we get the data. the call to get the sampleText changes where it looks for the data

Reply:
@bradflood It has been removed as well. All changes about the directories have been removed.